server/views/partials/message-content.html

```diff
@@ -3,34 +3,50 @@
     id: "text",
     name: "text",
     label: {
-      text: "Text message"
+      text: "Message text",
+      classes: "govuk-label--s"
     },
     hint: {
-      text: "Enter the message you wish to send."
+      text: "Enter the message to send."
     },
     maxlength: data.maxMessageLength,
     value: data.text,
     errorMessage: errors.text
   })
 }}
+
+{{
+  govukDetails({
```

Comment:
Added some help text (yet to be expanded in earnest) based on feedback.
(same as help for additional content)